### PR TITLE
Core/Loot: Implement CMSG_LOOT_MONEY, CMSG_LOOT_RELEASE, SMSG_COIN_REMOVED, SMSG_LOOT_MONEY_NOTIFY

### PR DESCRIPTION
**Pull request by cemak**

### CODE REVIEW

**Comment by xerkoss** [comment]
Nice, but don't change Money to Coins :D

**Comment by cemak** [comment]
coin because in
void Loot::BuildLootResponse(WorldPackets::Loot::LootResponse& packet, Player* viewer, PermissionTypes permission) const

packet.Coins = gold;

**Comment by Shauren** [comment]
What is the name of the packet class in client? coins or money? use the one that appears in client

**Comment by cemak** [comment]
Shauren, ok

File: src/server/game/Handlers/LootHandler.cpp [hunk 1]
```diff
@@ -210,11 +210,11 @@ void WorldSession::HandleLootMoneyOpcode(WorldPacket& /*recvData*/)
             if (Guild* guild = sGuildMgr->GetGuildById(player->GetGuildId()))
                 if (uint32 guildGold = CalculatePct(loot->gold, player->GetTotalAuraModifier(SPELL_AURA_DEPOSIT_BONUS_MONEY_IN_GUILD_BANK_ON_LOOT)))
                     guild->HandleMemberDepositMoney(this, guildGold, true);
-
-            WorldPacket data(SMSG_LOOT_MONEY_NOTIFY, 4 + 1);
-            data << uint32(loot->gold);
-            data << uint8(1);   // "You loot..."
-            SendPacket(&data);
+
```
Comment (Vincent-Michael) [inline src/server/game/Handlers/LootHandler.cpp]: whitespace

**Comment by Mogadischu** [comment]
It was correct before... It's CoinRemoved and LootMoneyNotify - not MoneyRemoved
Blame blizz for different naming :)

File: src/server/game/Handlers/LootHandler.cpp (hunk 1 above)
Comment (cemak) [inline src/server/game/Handlers/LootHandler.cpp]: ok

**Comment by cemak** [comment]
laught

**Comment by cemak** [comment]
Mogadischu, ok change

**Comment by cemak** [comment]
make re-pull

**Pull request closed without merging**